src/quo/components/browser/new_tab/component_spec.cljs

```diff
@@ -0,0 +1,40 @@
+(ns quo.components.browser.new-tab.component-spec
+  (:require [quo.components.browser.new-tab.view :as new-tab]
+            [quo.foundations.resources :as quo.resources]
+            [test-helpers.component :as h]))
+(def aave (quo.resources/get-dapp :aave))
+(h/describe "new-tab"
+  (h/test "render with light theme and blue customization color"
+    (let [theme :light
+          props {:customization-color :blue}]
+      (h/render-with-theme-provider [new-tab/view props] theme)
+      (h/has-style (h/query-by-test-id "new-tab")
+                   {:borderStyle     "dashed"
+                    :alignItems      "center"
+                    :backgroundColor "rgba(42,74,245,0.1)"
+                    :width           160
+                    :borderWidth     1
+                    :justifyContent  "center"
+                    :borderColor     "rgba(42,74,245,0.4)"
+                    :borderRadius    16
+                    :height          172
+                   })
+      (-> (h/expect (h/query-by-translation-text "New tab"))
+          (h/is-truthy))))
+  (h/test "render with dark theme and orange customization color"
+    (let [theme :dark
+          props {:customization-color :orange}]
+      (h/render-with-theme-provider [new-tab/view props] theme)
+      (h/has-style (h/query-by-test-id "new-tab")
```

Comment:
according to react native testing library it is better to use other queries, e.g text or accessibility label and only use a test id in the last case.

Reply:
Agree, but I tried searching by text and I couldn't find it for some reason. I even tested with a method to search for translated text, but couldn't make it work. I can add the accessibility label

Reply:
I see that you set accessibility label as an untranslated string, this doesn't seem right as accessibility should be translated. But I'll add it this way any way